src/Classes/Item.lua

```diff
@@ -63,6 +63,198 @@ function ItemClass:ResetInfluence()
+								break
+							end
+						end
+					end
+					if not found and not specialModifierFoundList[k] then
+						specialModifierFoundList[k] = true
+						ConPrintf("[%s] ENTRY: %s", tagName, k)
+					end
+				end
+			end
+		end
+		for _, dv in ipairs(v) do
+			if dv:lower():find(tag_name) then
+				local found_2 = false
+				if #v.weightKey > 0 then
+					for _,wk in ipairs(v.weightKey) do
+						if wk == slot_name then
+							-- this is useless if the modTags = { } (is empty)
+							if #v.modTags > 0 then
+								for _,tag in ipairs(v.modTags) do
+									if tag:lower() == tag_name then
+										found_2 = true
+										break
+									else
+										local excluded = false
+										-- if we have an exclusion pattern list for that tagName and itemSlotName
+										if data.itemTagSpecialExclusionPattern[tagName] and data.itemTagSpecialExclusionPattern[tagName][itemSlotName] then
+											-- iterate across the exclusion patterns
+											for _, specialMod in ipairs(data.itemTagSpecialExclusionPattern[tagName][itemSlotName]) do
+												-- and if the description matches pattern exclude it
+												if dv:lower():find(specialMod:lower()) then
+													excluded = true
+													break
+												end
+											end
+										end
+										if excluded then
+											found_2 = true
+											break
+										end
+									end
+								end
+								if not found_2 and not inverseModifierFoundList[k] then
+									inverseModifierFoundList[k] = true
+									ConPrintf("[%s] appears in desc but not in tags. [%s] %s", tag_name, k, dv)
+									break
+								end
+							end
+						end
+					end
+				else
+					-- this is useless if the modTags = { } (is empty)
+					if #v.modTags > 0 then
+						for _,tag in ipairs(v.modTags) do
+							if tag:lower() == tag_name then
+								found_2 = true
+								break
+							else
+								local excluded = false
+								-- if we have an exclusion pattern list for that tagName and itemSlotName
+								if data.itemTagSpecialExclusionPattern[tagName] and data.itemTagSpecialExclusionPattern[tagName][itemSlotName] then
+									-- iterate across the exclusion patterns
+									for _, specialMod in ipairs(data.itemTagSpecialExclusionPattern[tagName][itemSlotName]) do
+										-- and if the description matches pattern exclude it
+										if dv:lower():find(specialMod:lower()) then
+											excluded = true
+											break
+										end
+									end
+								end
+								if excluded then
+									found_2 = true
+									break
+								end
+							end
+						end
+						if not found_2 and not inverseModifierFoundList[k] then
+							inverseModifierFoundList[k] = true
+							ConPrintf("[%s] appears in desc but not in tags. [%s] %s", tag_name, k, dv)
+						end
+					end
+				end
+			end
+		end
+	end
+end
+
+-- Iterate over modifiers to see if specific substring is found (for conditional checking)
+function ItemClass:FindModifierSubstring(substring, itemSlotName)
```

Comment:
Luckily mod parsing is cached, but if that ever becomes not true or this type of mod we may want to watch out for performance of running through every modifier when parsing.